plugins/UsersManager/templates/index.twig

```diff
@@ -13,6 +13,7 @@
     filter-access-levels="{{ filterAccessLevels|json_encode }}"
     filter-status-levels="{{ statusAccessLevels|json_encode }}"
     activated-plugins="{{ activatedPlugins|json_encode }}"
+    user-edit-extensions="{{ userEditExtensions|json_encode }}"
```

Comment:
With the event approach as mentioned in another comment, this may not be needed at all.